### PR TITLE
[TS] LPS-159458

### PR DESCRIPTION
**Pull request by ericyanLr**
Hi Team,

https://issues.liferay.com/browse/LPS-159458

The goal of this PR is to restore the **'Read more'** link that was removed by [LPS-138008](https://issues.liferay.com/browse/LPS-138008).

**Proposed Changes**

- Restore **'Read more'** link that was removed by [LPS-138008](https://issues.liferay.com/browse/LPS-138008)
  - Looking at the ticket description in [LPS-138008](https://issues.liferay.com/browse/LPS-138008), the goal was to only remove the **'patch'** link, not the **'Read more'** link
- Retrieve WebDAV help message in one place, by utilizing the **DLAccessFromDesktopDisplayContext**
- Update link to use **Liferay Learn** article instead of **Help Center** article

Can you help verify if utilizing the **DLAccessFromDesktopDisplayContext** in the **info_panel_file_entry.jsp** is okay? My goal for this change was to consolidate the logic to retrieve the generated help message.

Please let me know if you have any questions.
Thanks!

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1347401_2438

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp [hunk 1]
```diff
@@ -289,14 +289,15 @@ long assetClassPK = DLAssetHelperUtil.getAssetClassPK(fileEntry, fileVersion);
...
+						String webDavHelpMessage = LanguageUtil.format(request, "webdav-help", "https://help.liferay.com/hc/en-us/articles/360028720352-Desktop-Access-to-Documents-and-Media", false);
 						String webDavURLInputId = liferayPortletResponse.getNamespace() + "webDavURLInput";
 						%>
 
 						<div class="form-group">
 							<label for="<%= webDavURLInputId %>">
 								<liferay-ui:message key='<%= TextFormatter.format("webDavURL", TextFormatter.K) %>' />
 
-								<liferay-ui:icon-help message='<%= LanguageUtil.get(request, "webdav-help") %>' />
+								<liferay-ui:icon-help message='<%= webDavHelpMessage %>' />
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp]: I'm not able to click the link.

Can you?

https://user-images.githubusercontent.com/47524751/182390940-ecf5fdaa-468c-495c-8c75-41b809f97da7.mp4

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp (hunk 1 above)
Comment (ericyanLr) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp]: Hi @AliciaGarciaGarcia ,

You are right, the tooltip does not provide a clickable link.

From my understanding, this is due to the usage of the **liferay-ui:icon-help** taglib; it looks like the tooltip is expected to only provide text.

But, the link does work from the Access from Desktop menu.

Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp]: if the link of the icon do not work, can you delete the "read more" of the tooltip? (or talk with FI to use a better component) 
It's confusing and do not gives information to the user.

Thanks!

Comment (ericyanLr) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp]: Hi @AliciaGarciaGarcia ,

That's a good point.

**Regarding:**
> if the link of the icon do not work, can you delete the "read more" of the tooltip?

Sure, should I create another language key: **webdav-help-tooltip**, which does not include the 'Read more' in the message?

Comment (ericyanLr) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp]: Hi @AliciaGarciaGarcia ,

I have sent a new PR with updated changes: https://github.com/liferay-lima/liferay-portal/pull/3325

**Comment by AliciaGarciaGarcia** [comment]
closed in favor of **https://github.com/liferay-lima/liferay-portal/pull/3325**

**Pull request closed without merging**